README.md

```diff
@@ -19,50 +19,70 @@ The OSHDB offers a distributed data base that splits storage and computation loa
+
+Because of it's size and variety, possibilities of working with OSM history data are limited and there exists a lack of an easy-to-use analysis software. A goal of the OSHDB is to make OSM data more accessible to researchers, data journalists, community members and other interested people.
 
 Central Concepts
 ----------------
 
-### Data Partitioning
+The OSHDB is designed to be appropriate for a large spectrum of potential use cases and is therefore built around the following central ideas and design goals:
+
+* _Lossless Information_: The full OSM history data set should be stored and be queryable by the OSHDB, including errorneous or partially incomplete data.
+* _Simple, Generic API_: Writing queries with the OSHDB should be simple, and intuitive, while at the same time flexbile and generic to allow a wide variety of analysis queries. 
+* _High Performance_: The OSM history data set is large and thus requires efficiency in the way the data is stored and in the way it can be accessed and processed. 
+* _Local and Distributed Deployment_: Analysis queries should scale well from data explorations of small regions up to global studies of the complete OSM data set.
+
+### Data Model
+
+The OSHDB uses it's own binary data format that encapsulates the OSM history data available from [planet.osm.org](https://planet.osm.org/) and is optimized for efficiently storage and access to the history data. In order to allow parallelized data processing, the OSHDB data model also includes a data partitioning scheme.
 
-* *(close to osm data structures)*
-* *(no lossy transformations/compressions)*
-* *(partially resolved references, to speed up typical operations like "geometry building")*
-* *(split into cells)*
+[![schematic overview of the OSHDB data model](documentation/manual/data-model.svg)](documentation/manual/data-model.md)
 
-### Data Processing/Access
+See the [data model](documentation/manual/data-model.md) section of the documentation for further information about the OSHDB data model.
 
-* *(rest-based api for common queries)*
-* *(oshdb-api: flexible, simple, map-reudce pattern, works on actual geometries)*
-* *(raw-data api: direct access to underlying raw osm data)*
+### API
+
+The OSHDB offers a flexible, simple and intuitive application programming interface that provides helpful abstractions on top of the offered OSM data entities. It provides different _views_ on the OSM history data that allow to either investigate the OSM data at specific points in time (as snapshots), or to investigate all contributions to the OSM data in their entirety. The OSHDB API also allows to filter the OSM data by arbitrary regions, time ranges and OSM properties such as tags, entity type, etc.
```

Comment:
... Application Programming Interface...

Reply:
also: views bold rather than italic

Reply:
I prefer to write “application programming interface” not capitalized, but good point about usage of italic with a sans serif font (https://practicaltypography.com/bold-or-italic.html)!